SparkPost/2web2ui
# Sa 1350 - Add aggregates summary section

### What Changed
 - Created new action reducer pair for getting aggregates.
 - Added the aggregate section below the summary chart.

### How To Test
 -  go to report builder, `/reports/summary` with Hibana enabled.
 -  select various metrics and check the section below the chart.
 - Make sure to select many metrics to check the wrapping.
### To Do
- [ ] Possibly some sorting for how each metric is presented.
